boltstream/settings.py

```diff
@@ -107,9 +106,14 @@
 # https://docs.djangoproject.com/en/2.0/ref/settings/#databases
 
 DATABASES = {
```

Comment:
I think we should try to be database-agnostic from Django's perspective, if we can.  We should probably leave this just as the default `ENV.db_url` and then export an environment from Docker like:

```
DATABASE_URL=postgres://boltstream:boltstream@postgres:5432/boltstream
```

This way people can still just run the Django app on the local computer with sqlite for dev if they want to.